### PR TITLE
ports/rp2: Enable SSL module with mbedtls.

### PR DESCRIPTION
**Pull request by iabdalkader**

### CODE REVIEW

File: ports/rp2/mpconfigport.h [hunk 1]
```diff
@@ -61,6 +61,7 @@
 #define MICROPY_OPT_COMPUTED_GOTO               (1)
 
 // Python internal features
+#define MICROPY_TRACKED_ALLOC                   (MICROPY_SSL_MBEDTLS)
```
Comment (Gadgetoid) [inline ports/rp2/mpconfigport.h:64]: Motion for `MICROPY_TRACKED_ALLOC` to be enabled generally. We have a few modules that it would be useful for.

**Comment by dpgeorge** [comment]
This feature is included as a commit in #8540.

File: ports/rp2/mpconfigport.h (hunk 1 above)
Comment (dpgeorge) [reply ports/rp2/mpconfigport.h:64]: It adds a small bit of overhead (an extra word for the GC to check) if not used.  Maybe better to wrap this in `#ifndef MICROPY_TRACKED_ALLOC` ..., and then a board can optionally enable it?

**Comment by iabdalkader** [comment]
> This feature is included as a commit in #8540.

Okay closing PR.

**Pull request closed without merging**